docker-compose.yml

```diff
@@ -95,6 +95,16 @@ services:
           cpus: '0.05'
           memory: 32M
 
+  system-prune:
```

Comment:
I would change the service name to something less "scary", like `cleanup` or `swarm-cleanup`. Other than that, looks great 🔥⭐️

Reply:
Well, you have the same scary part a few lines below in the command 😂